### ISSUE TITLE
[Screen Reader-Settings-Give feed back] Heading levels are not defined properly on the Give Feedback Screen.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2563]
**User Experience:**
Users with cognitive disabilities may rely on utilities to outline or summarize content based on headings, labels, and other landmarks. When these headings and labels are not descriptive of their related content, these utilities can work poorly and degrade the user’s ability to understand and navigate the product or service.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
Repro Steps:-
1-Navigate to url:- https://makecode.com/
2-Navigate to the Microbit section element and select Code control given on it.
3-Navigate to the Settings control lying in the header section on the page and select it.
4-Navigate to the Give feedback menu option listed under the Settings menu control and select it.
5-Navigate to the various controls lying in the pane opened and select Next.
6-Navigate to 'How likely..' page ,provide some input and select Next/Previous button control.
7-Navigate to 'Provide changes..' page ,provide some input and select Next/Previous button control.
8-Navigate in heading mode of narrator and verify the issue.

**Actual Result**
1.Heading levels are not defined properly on Give Feedback Screen.
2.Heading levels 1 and 4 are defined and heading level 2 is defined and not visible on the screen.

**Expected Result**
1.Heading levels should be defined properly on Give feedback screen.
2.Only two heading levels 1 and 2 should be defined(If two headings are on the screen) and no hidden heading should be defined.

**Note:**
Narrator in heading mode is not navigating after heading level 2.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}

**Suggested fix:**
In all the applicable forms we need to check below-
1-Headings in HTML range from H1 (the most important) to H6 (the least important). It’s best to reserve Heading1 (H1) on a web page for the title of that page.
2-Headings don’t need to decrease from 1 to 6 on every page. While you don’t need to use all of the headings, they should still be sequential.
For example, avoid skips from H2 to H4.

**Refer Attachment**
![mas 2 4 6_makecode_heading h2 is defined](https://user-images.githubusercontent.com/30459830/28705208-6e302a94-738c-11e7-8173-045de48688de.png)
[MAS 2.4.6_MakeCode_Headings are not defined properly.zip](https://github.com/Microsoft/pxt/files/1182183/MAS.2.4.6_MakeCode_Headings.are.not.defined.properly.zip)

**Comment by veler** [comment]
The give feedback website is an external website. We don't have the control on it. We cannot fix it.

**Comment by v-amik** [comment]
As per process, bug can't be closed until it get assigned to HCL for regression

**Comment by veler** [comment]
Needs to be forwarded to [Survey Monkey](https://www.surveymonkey.com).

**Comment by abchatra** [comment]
External survey monkey issue. Tracking bug on them filed.
